pilot/pkg/model/gateway.go

```diff
@@ -210,3 +212,23 @@ func IsPassThroughServer(server *networking.Server) bool {
 
 	return false
 }
+
+// convert ./host to currentNamespace/Host
+// */host to just host
+// */* to just *
```

Comment:
What's the intention of this?  We have many places using host fields, like VirtualService DestinationRule and ServiceEntry. 

While change this can cause confusions for all others too.

Reply:
There is no change to syntax of any of the configs. We are simply allowing people to specify ns/host in addition to the usual host. Specifying ns/host allows people to import a virtual service from a specific namespace.